src/kerchunk/gran_to_cat.py

```diff
@@ -0,0 +1,197 @@
+    data = ref["refs"][f"{dim}/0"][6:]
+    blosc = zarr.Blosc("zstd")
+    delta = zarr.Delta(dtype=np.float64)
+    decoders = [base64.b64decode, blosc.decode, delta.decode]
+    for decode in decoders:
+        data = decode(data)
+    return data
+    
+        
+def geom_from_refs(refs: Dict[str, Any]):
+    def maybe_scalar(g):
+        if isinstance(g, tuple):
+            return g[0]
+        return g
+    crs = []
+    n = len(refs)
+    bbox = dict(xmin=np.empty(n), ymin=np.empty(n),
+                xmax=np.empty(n), ymax=np.empty(n))
+    for i, ref in enumerate(refs.values()):
+        attrs = ujson.loads(ref["refs"][".zattrs"])
+        crs.append(attrs["crs_wkt"])
+        coords = {dim: coord_from_ref(ref, dim) for dim in ["x", "y"]}
+        for stat in ["min", "max"]:
+            for dim, coord in coords.items():
+                bbox[dim+stat][i] = getattr(coord, stat)()
+    geoms = pd.Series(shapely.box(**bbox), index=list(refs.keys()))
+    geoms = pd.concat([gpd.GeoSeries(geom, crs=maybe_scalar(g)).to_crs(epsg=4326)
+                       for g, geom in geoms.groupby(crs)])
+    return geoms
+        
+
+def refs_to_stac_item(ref: Dict[str, Any]) -> Dict[str, Any]:
+    """Convert kerchunk references JSON to STAC item JSON"""
+    @functools.lru_cache(maxsize=None)
+    def _crs_json_from_epsg(epsg):
+        return pyproj.CRS.from_epsg(epsg).to_json_dict()
+    uri = ref["refs"]["v/0.0"][0]
+    if not uri.startswith("s3://"):
+        uri = "s3://" + uri
+    name = os.path.basename(uri)
+    poly = geom_from_refs({name: ref}).values[0]
+    items = []
+    attrs = ujson.loads(ref["refs"][".zattrs"])
+    epsg = attrs["spatial_epsg"]
+    time = attrs["date_center"]
+    # dateutil can"t parse ISO times ending in "."
+    if time.endswith("."):
+        time += "0"
+    template = ujson.loads(ujson.dumps(STAC_TEMPLATE))
+    template["properties"]["datetime"] = time
+    template["id"] = os.path.basename(uri)
+    template["id"] = name
+    template["assets"] = {name: {"href": uri}}
+    reference_system = _crs_json_from_epsg(epsg)
+
+    x = coord_from_ref(ref, "x")
+    y = coord_from_ref(ref, "y")
+    item = xstac.xarray_to_stac(DuckDataset(ref),
+                                template,
+                                temporal_dimension=False,
+                                x_dimension="x",
+                                y_dimension="y",
+                                validate=False,
+                                reference_system=reference_system,
+                                x_step=x[1]-x[0],
+                                y_step=y[1]-y[0],
+                                y_extent=[y.min(), y.max()],
+                                x_extent=[x.min(), x.max()])
+    
+    item.geometry = ujson.loads(shapely.to_geojson(poly))
+    item.bbox = list(poly.bounds)
+    return item.to_dict()
+
+@click.command()
+@click.argument("path")
+@click.option("--s3_path", type=str, default=None)
```

Comment:
```suggestion
@click.option("--s3-path", type=str, default=None)
```
Typically CLI arguments are kabob-ed not snak-ed and click should handle turning `-` to `_` for the actual variable.